tests/test_jenkins.py

```diff
@@ -0,0 +1,77 @@
+import pytest
+
+
+@pytest.fixture()
+def AnsibleDefaults(Ansible):
+
+    """ Load default variables into dictionary.
+
+    Args:
+        Ansible - Requires the ansible connection backend.
+    """
+    return Ansible("include_vars", "./defaults/main.yml")["ansible_facts"]
+
+
+@pytest.fixture()
+def AnsibleFacts(Ansible):
+
+    """ Load ansible facts into a dictionary.
+
+    Args:
+        Ansible - Requires ansible connection backend.
+    """
+    return Ansible("setup")["ansible_facts"]
+
+
+def test_jenkins_package(Package, AnsibleDefaults):
+
+    """ Ensure jenkins package is both installed and at the proper major
+    version.
+
+    Args:
+        Package - module to determine package install status and version.
+        AnsibleDefaults - module to pull the target version of Jenkins.
+
+    Raises:
+        AssertionError if package is not installed or the wrong version.
+    """
+    jenkins = Package("jenkins")
+    assert jenkins.is_installed
+    jenkins_major_version = AnsibleDefaults["jenkins_major_version"]
+    assert jenkins.version.startswith(jenkins_major_version)
+
+
+def test_jenkins_service(Service, Sudo):
+
+    """ Ensure Jenkins service is enabled and running.
+
+    Args:
+        Service: module used to determine jenkins service status.
+        Sudo: module used for privilege elevation.
+
+    Raises:
+        AssertionError if service isn't enabled and running.
+    """
+
+    jenkins = Service("jenkins")
+    with Sudo():
```

Comment:
I was unable to run the tests without using `sudo` either as a module in the tests, or as a command line argument. Below is output from when I removed the `Sudo()` blocks from `test_jenkins.py` and  ran `molecule verify` with and without `sudo` escalation.

```
tnation@walnut:src/ansible-jenkins % molecule verify
--> Executing testinfra tests found in tests/.
============================= test session starts ==============================
platform darwin -- Python 2.7.10, pytest-2.9.2, py-1.4.31, pluggy-0.3.1
rootdir: /Users/tnation/Documents/projects/src/ansible-jenkins/tests, inifile:
plugins: xdist-1.15.0, testinfra-1.4.1
collected 3 itemss

tests/test_jenkins.py .F.

=================================== FAILURES ===================================
_______________ test_jenkins_service[ansible://ansible-jenkins] ________________

Service = <class 'testinfra.modules.base.UpstartService'>

    def test_jenkins_service(Service):
        """ Ensure Jenkins service is enabled and running.
    
        Args:
            Service: module used to determine jenkins service status.
            Sudo: module used for privilege elevation.
    
        Raises:
            AssertionError if service isn't enabled and running.
        """
    
        jenkins = Service("jenkins")
        assert jenkins.is_enabled
>       assert jenkins.is_running
E       assert <service jenkins>.is_running

tests/test_jenkins.py:54: AssertionError
====================== 1 failed, 2 passed in 2.88 seconds ======================
ERROR: 

  RAN: '/Users/tnation/.venvs/ansible-jenkins/bin/testinfra tests/test_jenkins.py --connection=ansible --ansible-inventory=.molecule/ansible_inventory'

```
```
tnation@walnut:src/ansible-jenkins % molecule verify --sudo      
--> Executing testinfra tests found in tests/.
============================= test session starts ==============================
platform darwin -- Python 2.7.10, pytest-2.9.2, py-1.4.31, pluggy-0.3.1
rootdir: /Users/tnation/Documents/projects/src/ansible-jenkins/tests, inifile:
plugins: xdist-1.15.0, testinfra-1.4.1
collected 3 itemss

tests/test_jenkins.py ...

=========================== 3 passed in 2.94 seconds ===========================
No serverspec tests found in spec/.
```